tests/test_server.py

```diff
@@ -77,6 +82,10 @@ def create_factory_dependency():
         cls.client = TestClient(app)
         cls.butler = _make_remote_butler(cls.client)
 
+        # Populate the test server.
+        server_butler.import_(filename=os.path.join(TESTDIR, "data", "registry", "base.yaml"))
+        server_butler.import_(filename=os.path.join(TESTDIR, "data", "registry", "datasets-uuid.yaml"))
+
```

Comment:
Doesn't matter yet but you may want to move this above the call to _make_remote_butler, in case we decide to read some stuff from the server during RemoteButler initialization.

Reply:
I'm hoping that it doesn't make a difference here either way because the clients won't be downloading caches of dataset type and collection before this runs will they?